skymp5-server/cpp/CMakeLists.txt

```diff
@@ -3,6 +3,17 @@ enable_testing()
 include(${CMAKE_SOURCE_DIR}/cmake/apply_default_settings.cmake)
 include(${CMAKE_SOURCE_DIR}/cmake/add_papyrus_library_ck.cmake)
 
+execute_process(
+  COMMAND git rev-parse HEAD
+  OUTPUT_VARIABLE GIT_COMMIT_ID
+  RESULT_VARIABLE GIT_RESULT
+  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+)
+string(STRIP "${GIT_COMMIT_ID}" GIT_COMMIT_ID)
+if(NOT "${GIT_RESULT}" STREQUAL "0")
+  message(WARNING "'git rev-parse HEAD' exited with '${GIT_RESULT}'")
```

Comment:
That's not fatal. We want empty password (will it work?) in such cases?

Reply:
In case user builds the project on a system with Git missing

Reply:
:thinking: Seems like really best option...

Does empty password mean 'no password'?

Reply:
BTW, I guess default installation of 'Git for Windows' doesn't add `git` binary dir to `PATH`, so it can be unnoticed (especially given we have other warnings which aren't fixed). Maybe option `-DSKYMP_DISABLE_PASSWORD=1` (suggested in another thread) will be a great fit here? I mean, explicitly failing build forcing to specify it if you have a reason for not installing `git` (or not adding it to `PATH`)